src/file_dialog.rs

```diff
@@ -1639,23 +1653,23 @@ impl FileDialog {
 
     /// Updates a device entry of a device list like "Devices" or "Removable Devices".
     fn ui_update_device_entry(&mut self, ui: &mut egui::Ui, device: &Disk) {
-        let label = match device.is_removable() {
-            true => format!(
-                "{}  {}",
+        let label = if device.is_removable() {
+            format!(
+                "{} {}",
```

Comment:
There should be two spaces. This is the label that is rendered in the UI. With a single space, the icons are too close to the text.

```suggestion
                "{}  {}",
```